test/Test.ml

```diff
@@ -4,7 +4,7 @@ let header fname =
   String.make 20 '-' ^ "[" ^ fname ^ "]" ^ String.make 20 '-' ^ "\n"
 
 let execute_file fname =
-  if String.equal (Filename.extension fname) ".cooltt" then
+  if String.equal (Filename.extension fname) ".cooltt" && fname <> "META.cooltt" then
```

Comment:
Taste it! I don't know why it's triggered in Docker.

Reply:
🥴🤦🏻‍♂️

Reply:
Feel free to add/suggest OCaml comments for non-genie mortals.